### PR TITLE
Added troubleshooting checklist

### PR DESCRIPTION
**Pull request by ravenac95**

### CODE REVIEW

File: src/operators/troubleshooting.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Troubleshooting
 
+## BEFORE YOUR TROUBLESHOOT CHECKLIST
```
Comment (ravenac95) [inline src/operators/troubleshooting.md]: Going to make this a giant warning block

File: src/operators/troubleshooting.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+
+* Check that your current binary version is the latest listed on the [current
+  testnet parameters](./current-testnet-parameters.md)
+  * Check the version on your localhost using `oasis-node --version`
+  * Check the version on your server using `oasis-node --version`
+* If upgrading, make sure that you've wiped state (unless that is explicitly not
+  required)
+* If you're doing anything with the entity:
+  * Do you have the latest genesis?
+  * Do you have the correct private key (or ledger device).
```
Comment (peterjgilbert) [inline src/operators/troubleshooting.md]:
```suggestion
  * Do you have the correct private key (or Ledger device).
```

File: src/operators/troubleshooting.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+  * If you're signing a transaction:
+    * Do you have enough of an account balance to make the transaction?
```
Comment (peterjgilbert) [inline src/operators/troubleshooting.md]:
```suggestion
    * Do you sufficient account balance to make the transaction?
```

File: src/operators/troubleshooting.md [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+      * Run `oasis-node stake account info`
+    * Are you using the correct nonce?
+      * Run `oasis-node stake account info`
+* If you're generating a transaction:
+  * Do you have the latest genesis?
+* If you're submitting a transaction:
+  * Do you have the latest genesis?
+  * Is your node synced? The transaction will fail to run properly
```
Comment (peterjgilbert) [inline src/operators/troubleshooting.md]:
```suggestion
  * Is your node synced? If not, the transaction will fail to run properly
```

**Pull request merged**